fe/fe-core/src/main/java/com/starrocks/sql/optimizer/ConstantOperatorUtils.java

```diff
@@ -0,0 +1,60 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.optimizer;
+
+import com.starrocks.sql.optimizer.operator.scalar.ConstantOperator;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+/**
+ * TYPE            |  JAVA_TYPE
+ * TYPE_INVALID    |    null
+ * TYPE_NULL       |    null
+ * TYPE_BOOLEAN    |    boolean
+ * TYPE_TINYINT    |    byte
+ * TYPE_SMALLINT   |    short
+ * TYPE_INT        |    int
+ * TYPE_BIGINT     |    long
+ * TYPE_LARGEINT   |    BigInteger
+ * TYPE_FLOAT      |    double
+ * TYPE_DOUBLE     |    double
+ * TYPE_DATE       |    LocalDateTime
+ * TYPE_DATETIME   |    LocalDateTime
+ * TYPE_TIME       |    LocalDateTime
+ * TYPE_DECIMAL    |    BigDecimal
+ * TYPE_DECIMALV2  |    BigDecimal
+ * TYPE_VARCHAR    |    String
+ * TYPE_CHAR       |    String
+ * TYPE_HLL        |    NOT_SUPPORT
+ * TYPE_BITMAP     |    NOT_SUPPORT
+ * TYPE_PERCENTILE |    NOT_SUPPORT
+ */
+public class ConstantOperatorUtils {
+
+    public static double getDoubleValue(ConstantOperator constantOperator) {
```

Comment:
You can use function doubleValueFromConstant in the ExpressionStatisticCalculator

Reply:
But function doubleValueFromConstant is private. Can i edit it?